.gitignore

```
@@ -39,3 +39,4 @@ nosetests.xml
 
 # rope
 .ropeproject
+/.vs
```

```suggestion
.vs
```

I don't think the `/` is needed.